process/block/baseProcess.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
```

Comment:
please optimize as well `func (mp *metaProcessor) checkProofsForShardData(header *block.MetaBlock)` to request needed blocks and verify the shard data in parallel